src/widgetastic_patternfly4/ouia.py

```diff
@@ -155,3 +156,7 @@ class TextInput(BaseOuiaTextInput):
 
 class ClipboardCopy(BaseClipboardCopy, OUIAGenericWidget):
     OUIA_COMPONENT_TYPE = "PF4/ClipboardCopy"
+
+
+class ClipboardCopyCompact(BaseClipboardCopyCompact, OUIAGenericWidget):
+    OUIA_COMPONENT_TYPE = "PF4/ClipboardCopy"
```

Comment:
I think this should actually be considered an `Inline` widget class instead of Compact, and we could also have an `Expanded` class. This is due to the use of `pf-m-inline` and `pf-m-expanded` classes that differentiate the component divs that determine the root locator.

Here your approach to subclass these different ClipboardCopy component variations breaks down a bit, because they all share a common OUIA component type, and the OUIAGenericWidget locator will determine the locator.

The alternative is to write xpath expressions that capture both 'modes' of locating the text/button components, or to write the widget functions to rely on a property that checks for the `inline` or `expanded` class in the div to dictate behavior for read/fill/click.